src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+const clickPromis = new Promise((resolve, reject) => {
+  logo.addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+
+  setTimeout(() => {
+    // eslint-disable-next-line prefer-promise-reject-errors
+    reject('Promise was rejected!');
+  }, 3000);
+});
```

Comment:
Your promise will always be rejected after 3 seconds, regardless of whether the logo was clicked or not. This is because the setTimeout function will always execute after the specified delay. To fix this, you should clear the timeout when the logo is clicked. You can do this by calling clearTimeout() and passing the ID value returned by setTimeout().